建议加个选项取消掉部分日志记录

- [x] 你是否在现有 [Issue列表](/docmirror/dev-sidecar/issues) 中搜索过相同问题，但未找到？

### Ⅰ. 请说明操作系统及DS的版本号：

1. 操作系统：Windows 11
2. DS版本号：2.0.0-RC4

### Ⅱ. 问题描述：
server.log在一天内增加到7GB

### Ⅲ. 期望的结果：
建议加个选项取消掉所有请求失败和部分info日志

### Ⅳ. 如何复现问题？
1.打开软件 & 浏览器
2.等待
(可能是我的网络问题)

### Ⅴ. 请提供相关的错误日志，尽可能的详细：（日志文件在 `${user.home}/.dev-sidecar/logs/` 目录下）

<details>
<summary>点击查看日志</summary>

```log
connect ETIMEDOUT 1.1.1.1:443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
  errno: -4039,
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '1.1.1.1',
  port: 443
}
[2025-01-28T10:00:38.724] [WARN] server - DNS query error: alive.github.com, dns: cloudflare, dnsServer: https://1.1.1.1/dns-query, cost: 21041 ms, error: Error: connect ETIMEDOUT 1.1.1.1:443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
  errno: -4039,
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '1.1.1.1',
  port: 443
}
[2025-01-28T10:00:38.740] [WARN] server - DNS query error: alive.github.com, dns: rubyfish, dnsServer: https://rubyfish.cn/dns-query, cost: 21057 ms, error: Error: connect ETIMEDOUT 175.24.154.66:443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
  errno: -4039,
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '175.24.154.66',
  port: 443
}
[2025-01-28T10:00:38.740] [INFO] server - [speed] alive.github.com ➜ ip-list: [ { host: '140.82.112.26', port: 443, dns: 'safe360' } ]
[2025-01-28T10:00:38.741] [WARN] server - DNS query error: alive.github.com, dns: rubyfish, dnsServer: https://rubyfish.cn/dns-query, cost: 21057 ms, error: Error: connect ETIMEDOUT 175.24.154.66:443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
  errno: -4039,
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '175.24.154.66',
  port: 443
}
[2025-01-28T10:00:38.741] [INFO] server - [speed] alive.github.com ➜ ip-list: [ { host: '140.82.112.26', port: 443, dns: 'safe360' } ]
[2025-01-28T10:00:40.383] [ERROR] server - cltSocket error:   update.googleapis.com:443, errorMsg: read ECONNRESET
[2025-01-28T10:00:47.654] [INFO] server - 不拦截请求，直连目标服务器: hcfy.ai:443, headers: {"host":"hcfy.ai:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:00:48.178] [ERROR] server - 直连失败: optimizationguide-pa.googleapis.com:443, cost: 21061 ms, errorMsg: connect ETIMEDOUT 172.217.14.234:443
 Error: connect ETIMEDOUT 172.217.14.234:443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
  errno: -4039,
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '172.217.14.234',
  port: 443
}
......
*":{"proxy":"fonts.loli.net","test":"https://fonts.googleapis.com/css?family=Oswald"}} }
[2025-01-28T10:01:33.620] [INFO] server - getServerPromise, hostname: fonts.googleapis.com:443, ssl: true, protocol: https
[2025-01-28T10:01:33.621] [INFO] server - 【CreateFakeServer】hostname: fonts.googleapis.com:443, ssl: true, protocol: https
[2025-01-28T10:01:33.621] [INFO] server - 【CreateFakeCertificate】dnsName: *.googleapis.com, hostname: fonts.googleapis.com:443
[2025-01-28T10:01:33.981] [INFO] server - ----- fakeServer connect: 127.0.0.1:51236 ➜ fonts.googleapis.com:443 -----
[2025-01-28T10:01:33.983] [INFO] server - Proxy connect start: 127.0.0.1:51236
[2025-01-28T10:01:33.984] [INFO] server - fakeServer SNICallback: fonts.googleapis.com:443
[2025-01-28T10:01:33.994] [INFO] server - 创建 HttpsAgent 成功, timeoutConfig: { timeout: 20000, keepAliveTimeout: 30000 } , verifySsl: true
[2025-01-28T10:01:33.995] [INFO] server - matchHostname-all: get interceptOpts: 'fonts.googleapis.com': {".*":{"proxy":"fonts.loli.net","test":"https://fonts.googleapis.com/css?family=Oswald"}}
[2025-01-28T10:01:33.998] [INFO] server - proxy intercept: hostname: fonts.googleapis.com, target：https://fonts.loli.net/css2?family=Albert+Sans:ital,wght@0,100..900;1,100..900&display=swap
[2025-01-28T10:01:33.999] [INFO] server - 发起代理请求: GET ➜ https://fonts.loli.net:443/css2?family=Albert+Sans:ital,wght@0,100..900;1,100..900&display=swap  , headers: {"host":"fonts.loli.net","connection":"keep-alive","sec-ch-ua-platform":"\"Windows\"","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0","sec-ch-ua":"\"Not A(Brand\";v=\"8\", \"Chromium\";v=\"132\", \"Microsoft Edge\";v=\"132\"","sec-ch-ua-mobile":"?0","accept":"text/css,*/*;q=0.1","sec-fetch-site":"cross-site","sec-fetch-mode":"no-cors","sec-fetch-dest":"style","referer":"https://47.116.124.28:3075/","accept-encoding":"gzip, deflate, br, zstd","accept-language":"zh-CN,zh;q=0.9,en;q=0.8,en-GB;q=0.7,en-US;q=0.6"}
[2025-01-28T10:01:34.000] [INFO] server - 域名 fonts.loli.net 在DNS中未配置
[2025-01-28T10:01:34.000] [INFO] server - getRequestConfig: fonts.loli.net:443, undefined
[2025-01-28T10:01:34.858] [INFO] server - checkServerIdentity: fonts.loli.net, CN: fonts.loli.net, C: US, ST: undefined, bits: 256
[2025-01-28T10:01:35.101] [INFO] server - 不拦截请求，直连目标服务器: unpkg.com:443, headers: {"host":"unpkg.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:01:35.406] [INFO] server - 代理请求返回: 【200】GET ➜ https://fonts.loli.net:443/css2?family=Albert+Sans:ital,wght@0,100..900;1,100..900&display=swap, cost: 1407 ms
[2025-01-28T10:01:41.519] [INFO] server - 不拦截请求，直连目标服务器: gator.volces.com:443, headers: {"host":"gator.volces.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:01:41.785] [INFO] server - 不拦截请求，直连目标服务器: gator.volces.com:443, headers: {"host":"gator.volces.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:02:10.329] [INFO] server - 不拦截请求，直连目标服务器: www.bing.com:443, headers: {"host":"www.bing.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:03:14.881] [INFO] server - 不拦截请求，直连目标服务器: optimizationguide-pa.googleapis.com:443, headers: {"host":"optimizationguide-pa.googleapis.com:443","proxy-connection":"keep-alive","user-agent":"DeepL/25.1.4.15077 (Microsoft Windows NT 10.0.22631.0; x64)"}
[2025-01-28T10:03:35.955] [ERROR] server - 直连失败: optimizationguide-pa.googleapis.com:443, cost: 21074 ms, errorMsg: connect ETIMEDOUT 142.251.215.234:443
 Error: connect ETIMEDOUT 142.251.215.234:443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
  errno: -4039,
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '142.251.215.234',
  port: 443
}
[2025-01-28T10:04:13.859] [ERROR] server - 直连失败: www.bing.com:443, cost: 123530 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2025-01-28T10:05:53.918] [INFO] server - 不拦截请求，直连目标服务器: optimizationguide-pa.googleapis.com:443, headers: {"host":"optimizationguide-pa.googleapis.com:443","proxy-connection":"keep-alive","user-agent":"DeepL/25.1.4.15077 (Microsoft Windows NT 10.0.22631.0; x64)"}
[2025-01-28T10:06:16.114] [ERROR] server - 直连失败: optimizationguide-pa.googleapis.com:443, cost: 22194 ms, errorMsg: connect ETIMEDOUT 142.251.215.234:443
 Error: connect ETIMEDOUT 142.251.215.234:443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
  errno: -4039,
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '142.251.215.234',
  port: 443
}
...
[2025-01-28T10:15:00.588] [ERROR] server - 直连失败: aefd.nelreports.net:443, cost: 2 ms, errorMsg: getaddrinfo ENOTFOUND aefd.nelreports.net
 Error: getaddrinfo ENOTFOUND aefd.nelreports.net
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:71:26) {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'aefd.nelreports.net'
}
[2025-01-28T10:15:00.591] [ERROR] server - 直连失败: aefd.nelreports.net:443, cost: 4 ms, errorMsg: getaddrinfo ENOTFOUND aefd.nelreports.net
 Error: getaddrinfo ENOTFOUND aefd.nelreports.net
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:71:26) {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'aefd.nelreports.net'
}
[2025-01-28T10:15:00.661] [INFO] server - 不拦截请求，直连目标服务器: alive.github.com:443, headers: {"host":"alive.github.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:15:00.661] [INFO] server - matchHostname: get dns providerName: 'alive.github.com' -> { "^.*\.github\.com$": "quad9" }
[2025-01-28T10:15:00.661] [INFO] server - ----- connect: alive.github.com, use alive ip from dns 'safe360': 140.82.112.26, target: alive.github.com:443 -----
[2025-01-28T10:15:00.662] [ERROR] server - 直连失败: alive.github.com:443, cost: 1 ms, errorMsg: connect EHOSTUNREACH 140.82.112.26:443
 Error: connect EHOSTUNREACH 140.82.112.26:443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
  errno: -4073,
  code: 'EHOSTUNREACH',
  syscall: 'connect',
  address: '140.82.112.26',
  port: 443
}
[2025-01-28T10:15:01.696] [INFO] server - 不拦截请求，直连目标服务器: alive.github.com:443, headers: {"host":"alive.github.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:15:01.696] [INFO] server - matchHostname: get dns providerName: 'alive.github.com' -> { "^.*\.github\.com$": "quad9" }
[2025-01-28T10:15:01.696] [INFO] server - ----- connect: alive.github.com, use alive ip from dns 'safe360': 140.82.112.26, target: alive.github.com:443 -----
.....
[2025-01-28T10:10:24.617] [INFO] server - 不拦截请求，直连目标服务器: www.bing.com:443, headers: {"host":"www.bing.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:24.854] [INFO] server - 不拦截请求，直连目标服务器: www.bing.com:443, headers: {"host":"www.bing.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:25.011] [INFO] server - 不拦截请求，直连目标服务器: www.bing.com:443, headers: {"host":"www.bing.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:25.174] [INFO] server - 不拦截请求，直连目标服务器: www.bing.com:443, headers: {"host":"www.bing.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:25.401] [INFO] server - 不拦截请求，直连目标服务器: www.bing.com:443, headers: {"host":"www.bing.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:26.327] [INFO] server - 不拦截请求，直连目标服务器: ts2.cn.mm.bing.net:443, headers: {"host":"ts2.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:26.336] [INFO] server - 不拦截请求，直连目标服务器: www.bing.com:443, headers: {"host":"www.bing.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:26.366] [INFO] server - 不拦截请求，直连目标服务器: th.bing.com:443, headers: {"host":"th.bing.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:26.366] [INFO] server - 不拦截请求，直连目标服务器: sb.scorecardresearch.com:443, headers: {"host":"sb.scorecardresearch.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:26.499] [INFO] server - 不拦截请求，直连目标服务器: ts2.cn.mm.bing.net:443, headers: {"host":"ts2.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:26.499] [INFO] server - 不拦截请求，直连目标服务器: ts2.cn.mm.bing.net:443, headers: {"host":"ts2.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:26.499] [INFO] server - 不拦截请求，直连目标服务器: ts2.cn.mm.bing.net:443, headers: {"host":"ts2.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:26.500] [INFO] server - 不拦截请求，直连目标服务器: ts2.cn.mm.bing.net:443, headers: {"host":"ts2.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:26.500] [INFO] server - 不拦截请求，直连目标服务器: ts2.cn.mm.bing.net:443, headers: {"host":"ts2.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:26.943] [INFO] server - 不拦截请求，直连目标服务器: ts2.cn.mm.bing.net:443, headers: {"host":"ts2.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:26.948] [ERROR] server - cltSocket error:   ts2.cn.mm.bing.net:443, errorMsg: write ECONNABORTED
[2025-01-28T10:10:26.981] [INFO] server - 不拦截请求，直连目标服务器: r.bing.com:443, headers: {"host":"r.bing.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:26.982] [INFO] server - 不拦截请求，直连目标服务器: r.bing.com:443, headers: {"host":"r.bing.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:26.982] [INFO] server - 不拦截请求，直连目标服务器: th.bing.com:443, headers: {"host":"th.bing.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:26.982] [INFO] server - 不拦截请求，直连目标服务器: th.bing.com:443, headers: {"host":"th.bing.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:27.770] [INFO] server - 不拦截请求，直连目标服务器: ts3.cn.mm.bing.net:443, headers: {"host":"ts3.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:27.771] [INFO] server - 不拦截请求，直连目标服务器: ts1.cn.mm.bing.net:443, headers: {"host":"ts1.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:27.772] [INFO] server - 不拦截请求，直连目标服务器: ts4.cn.mm.bing.net:443, headers: {"host":"ts4.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:27.772] [INFO] server - 不拦截请求，直连目标服务器: ts2.cn.mm.bing.net:443, headers: {"host":"ts2.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:27.909] [INFO] server - 不拦截请求，直连目标服务器: ts2.cn.mm.bing.net:443, headers: {"host":"ts2.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:27.909] [INFO] server - 不拦截请求，直连目标服务器: ts2.cn.mm.bing.net:443, headers: {"host":"ts2.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:27.910] [INFO] server - 不拦截请求，直连目标服务器: ts2.cn.mm.bing.net:443, headers: {"host":"ts2.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:27.958] [INFO] server - 不拦截请求，直连目标服务器: ts4.cn.mm.bing.net:443, headers: {"host":"ts4.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:27.959] [INFO] server - 不拦截请求，直连目标服务器: ts4.cn.mm.bing.net:443, headers: {"host":"ts4.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:27.960] [INFO] server - 不拦截请求，直连目标服务器: ts3.cn.mm.bing.net:443, headers: {"host":"ts3.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:27.960] [INFO] server - 不拦截请求，直连目标服务器: ts3.cn.mm.bing.net:443, headers: {"host":"ts3.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:28.075] [INFO] server - 不拦截请求，直连目标服务器: ts1.cn.mm.bing.net:443, headers: {"host":"ts1.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:28.076] [INFO] server - 不拦截请求，直连目标服务器: ts1.cn.mm.bing.net:443, headers: {"host":"ts1.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:44.239] [INFO] server - 不拦截请求，直连目标服务器: ts1.cn.mm.bing.net:443, headers: {"host":"ts1.cn.mm.bing.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:44.293] [ERROR] server - 直连失败: r.bing.com:443, cost: 17312 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2025-01-28T10:10:49.107] [INFO] server - 不拦截请求，直连目标服务器: clink-api.gitcode.com:443, headers: {"host":"clink-api.gitcode.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:52.041] [INFO] server - 不拦截请求，直连目标服务器: aefd.nelreports.net:443, headers: {"host":"aefd.nelreports.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:52.041] [INFO] server - 不拦截请求，直连目标服务器: aefd.nelreports.net:443, headers: {"host":"aefd.nelreports.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:10:52.588] [ERROR] server - 直连失败: th.bing.com:443, cost: 25606 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2025-01-28T10:10:52.605] [ERROR] server - 直连失败: r.bing.com:443, cost: 25622 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2025-01-28T10:11:09.668] [INFO] server - 不拦截请求，直连目标服务器: optimizationguide-pa.googleapis.com:443, headers: {"host":"optimizationguide-pa.googleapis.com:443","proxy-connection":"keep-alive","user-agent":"DeepL/25.1.4.15077 (Microsoft Windows NT 10.0.22631.0; x64)"}
[2025-01-28T10:11:30.749] [ERROR] server - 直连失败: optimizationguide-pa.googleapis.com:443, cost: 21081 ms, errorMsg: connect ETIMEDOUT 172.217.14.202:443
 Error: connect ETIMEDOUT 172.217.14.202:443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
  errno: -4039,
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '172.217.14.202',
  port: 443
}
[2025-01-28T10:11:52.852] [ERROR] server - 直连失败: th.bing.com:443, cost: 86486 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2025-01-28T10:11:52.855] [ERROR] server - 直连失败: www.bing.com:443, cost: 87680 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2025-01-28T10:11:52.859] [ERROR] server - 直连失败: www.bing.com:443, cost: 87848 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2025-01-28T10:11:52.860] [ERROR] server - 直连失败: www.bing.com:443, cost: 88006 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2025-01-28T10:11:52.862] [ERROR] server - 直连失败: www.bing.com:443, cost: 87461 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2025-01-28T10:11:52.864] [ERROR] server - 直连失败: th.bing.com:443, cost: 85882 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2025-01-28T10:11:52.867] [ERROR] server - 直连失败: www.bing.com:443, cost: 88354 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2025-01-28T10:12:30.131] [ERROR] server - 直连失败: www.bing.com:443, cost: 125514 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2025-01-28T10:12:33.562] [ERROR] server - 直连失败: www.bing.com:443, cost: 127226 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2025-01-28T10:13:10.957] [INFO] server - 不拦截请求，直连目标服务器: www.bing.com:443, headers: {"host":"www.bing.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:13:50.816] [INFO] server - 不拦截请求，直连目标服务器: optimizationguide-pa.googleapis.com:443, headers: {"host":"optimizationguide-pa.googleapis.com:443","proxy-connection":"keep-alive","user-agent":"DeepL/25.1.4.15077 (Microsoft Windows NT 10.0.22631.0; x64)"}
[2025-01-28T10:14:11.906] [ERROR] server - 直连失败: optimizationguide-pa.googleapis.com:443, cost: 21090 ms, errorMsg: connect ETIMEDOUT 142.250.69.202:443
 Error: connect ETIMEDOUT 142.250.69.202:443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
  errno: -4039,
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '142.250.69.202',
  port: 443
}
[2025-01-28T10:14:43.179] [ERROR] server - 直连失败: www.bing.com:443, cost: 92222 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2025-01-28T10:15:00.586] [INFO] server - 不拦截请求，直连目标服务器: aefd.nelreports.net:443, headers: {"host":"aefd.nelreports.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:15:00.587] [INFO] server - 不拦截请求，直连目标服务器: aefd.nelreports.net:443, headers: {"host":"aefd.nelreports.net:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:15:00.588] [ERROR] server - 直连失败: aefd.nelreports.net:443, cost: 2 ms, errorMsg: getaddrinfo ENOTFOUND aefd.nelreports.net
 Error: getaddrinfo ENOTFOUND aefd.nelreports.net
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:71:26) {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'aefd.nelreports.net'
}
[2025-01-28T10:15:00.591] [ERROR] server - 直连失败: aefd.nelreports.net:443, cost: 4 ms, errorMsg: getaddrinfo ENOTFOUND aefd.nelreports.net
 Error: getaddrinfo ENOTFOUND aefd.nelreports.net
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:71:26) {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'aefd.nelreports.net'
}
[2025-01-28T10:15:00.661] [INFO] server - 不拦截请求，直连目标服务器: alive.github.com:443, headers: {"host":"alive.github.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:15:00.661] [INFO] server - matchHostname: get dns providerName: 'alive.github.com' -> { "^.*\.github\.com$": "quad9" }
[2025-01-28T10:15:00.661] [INFO] server - ----- connect: alive.github.com, use alive ip from dns 'safe360': 140.82.112.26, target: alive.github.com:443 -----
[2025-01-28T10:15:00.662] [ERROR] server - 直连失败: alive.github.com:443, cost: 1 ms, errorMsg: connect EHOSTUNREACH 140.82.112.26:443
 Error: connect EHOSTUNREACH 140.82.112.26:443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
  errno: -4073,
  code: 'EHOSTUNREACH',
  syscall: 'connect',
  address: '140.82.112.26',
  port: 443
}
[2025-01-28T10:15:01.696] [INFO] server - 不拦截请求，直连目标服务器: alive.github.com:443, headers: {"host":"alive.github.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:15:01.696] [INFO] server - matchHostname: get dns providerName: 'alive.github.com' -> { "^.*\.github\.com$": "quad9" }
[2025-01-28T10:15:01.696] [INFO] server - ----- connect: alive.github.com, use alive ip from dns 'safe360': 140.82.112.26, target: alive.github.com:443 -----
[2025-01-28T10:15:38.857] [INFO] server - 不拦截请求，直连目标服务器: hcfy.ai:443, headers: {"host":"hcfy.ai:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:16:09.621] [INFO] server - 不拦截请求，直连目标服务器: www.bing.com:443, headers: {"host":"www.bing.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
[2025-01-28T10:16:32.637] [INFO] server - 不拦截请求，直连目标服务器: optimizationguide-pa.googleapis.com:443, headers: {"host":"optimizationguide-pa.googleapis.com:443","proxy-connection":"keep-alive","user-agent":"DeepL/25.1.4.15077 (Microsoft Windows NT 10.0.22631.0; x64)"}
[2025-01-28T10:16:54.036] [ERROR] server - 直连失败: optimizationguide-pa.googleapis.com:443, cost: 21399 ms, errorMsg: connect ETIMEDOUT 142.250.69.202:443
 Error: connect ETIMEDOUT 142.250.69.202:443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
  errno: -4039,
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '142.250.69.202',
  port: 443
}
[2025-01-28T10:17:00.392] [ERROR] server - 直连失败: www.bing.com:443, cost: 50771 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2025-01-28T10:17:09.684] [INFO] server - 不拦截请求，直连目标服务器: www.bing.com:443, headers: {"host":"www.bing.com:443","proxy-connection":"keep-alive","user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36 Edg/132.0.0.0"}
```
</details>

### Ⅵ. 有必要时，请提供 `${user.home}/.dev-sidecar/running.json` 文件内容：

<!-- 请将 'running.json' 文件的内容粘贴在这里，方便我们排查问题是否由配置错误导致。 -->

<details>
<summary>点击查看运行参数</summary>

```json

```
</details>

--- a/packages/core/src/config/index.js
+++ b/packages/core/src/config/index.js
@@ -37,6 +37,8 @@ const defaultConfig = {
     // 日志相关配置
     logFileSavePath: path.join(configLoader.getUserBasePath(), '/logs'), // 日志文件保存路径
     keepLogFileCount: 15, // 保留日志文件数
+    maxLogFileSize: 0, // 最大日志文件大小
+    maxLogFileSizeUnit: 'GB', // 最大日志文件大小单位
   },
   server: {
     enabled: true,

--- a/packages/core/src/utils/util.logger.js
+++ b/packages/core/src/utils/util.logger.js
@@ -23,12 +23,14 @@ function getDefaultConfigBasePath () {
 const basePath = getDefaultConfigBasePath()
 
 // 通用日志配置
+const maxLogSizeMultiple = 1024 * 1024 * (configFromFiles.app.maxLogFileSizeUnit === 'GB' ? 1024 : 1) // 目前单位只有GB和MB
 const appenderConfig = {
   type: 'file',
   pattern: 'yyyy-MM-dd',
   compress: true, // 压缩日志文件
   keepFileExt: true, // 保留日志文件扩展名为 .log
   backups: configFromFiles.app.keepLogFileCount, // 保留日志文件数
+  maxLogSize: (configFromFiles.app.maxLogFileSize || 0) * maxLogSizeMultiple,
 }
 
 let log = null

--- a/packages/core/src/utils/util.logger.js
+++ b/packages/core/src/utils/util.logger.js
@@ -23,14 +23,13 @@ function getDefaultConfigBasePath () {
 const basePath = getDefaultConfigBasePath()
 
 // 通用日志配置
-const maxLogFileSizeMultiple = 1024 * 1024 * (configFromFiles.app.maxLogFileSizeUnit === 'GB' ? 1024 : 1) // 目前单位只有GB和MB
 const appenderConfig = {
   type: 'file',
   pattern: 'yyyy-MM-dd',
   compress: true, // 压缩日志文件
   keepFileExt: true, // 保留日志文件扩展名为 .log
   backups: configFromFiles.app.keepLogFileCount, // 保留日志文件数
-  maxLogSize: (configFromFiles.app.maxLogFileSize || 0) * maxLogFileSizeMultiple,
+  maxLogSize: (configFromFiles.app.maxLogFileSize || 0) * 1024 * 1024 * (configFromFiles.app.maxLogFileSizeUnit === 'GB' ? 1024 : 1), // 目前单位只有GB和MB
 }
 
 let log = null

--- a/packages/core/src/utils/util.logger.js
+++ b/packages/core/src/utils/util.logger.js
@@ -23,14 +23,14 @@ function getDefaultConfigBasePath () {
 const basePath = getDefaultConfigBasePath()
 
 // 通用日志配置
-const maxLogSizeMultiple = 1024 * 1024 * (configFromFiles.app.maxLogFileSizeUnit === 'GB' ? 1024 : 1) // 目前单位只有GB和MB
+const maxLogFileSizeMultiple = 1024 * 1024 * (configFromFiles.app.maxLogFileSizeUnit === 'GB' ? 1024 : 1) // 目前单位只有GB和MB
 const appenderConfig = {
   type: 'file',
   pattern: 'yyyy-MM-dd',
   compress: true, // 压缩日志文件
   keepFileExt: true, // 保留日志文件扩展名为 .log
   backups: configFromFiles.app.keepLogFileCount, // 保留日志文件数
-  maxLogSize: (configFromFiles.app.maxLogFileSize || 0) * maxLogSizeMultiple,
+  maxLogSize: (configFromFiles.app.maxLogFileSize || 0) * maxLogFileSizeMultiple,
 }
 
 let log = null

--- a/packages/gui/src/view/pages/setting.vue
+++ b/packages/gui/src/view/pages/setting.vue
@@ -13,6 +13,16 @@ export default {
       reloadLoading: false,
       urlBackup: null,
       personalUrlBackup: null,
+      maxLogFileSizeUnit: [
+        {
+          label: 'GB',
+          value: 'GB',
+        },
+        {
+          label: 'MB',
+          value: 'MB',
+        },
+      ],
     }
   },
   methods: {
@@ -458,10 +468,22 @@ export default {
           注意：原目录中的文件不会自动转移到新的目录，请自行转移或删除。
         </div>
       </a-form-item>
+      <a-form-item label="最大日志文件大小" :label-col="labelCol" :wrapper-col="wrapperCol">
+        <a-input-number v-model="config.app.maxLogFileSize" :step="1" :min="0" />
+        <a-select v-model="config.app.maxLogFileSizeUnit" class="md-ml-5">
+          <a-select-option v-for="(item) of maxLogFileSizeUnit" :key="item.value" :value="item.value">
+            {{ item.label }}
+          </a-select-option>
+        </a-select>
+        <div class="form-help">
+          修改后，重启DS才生效！<br>
+          单个日志文件的大小限制，达到限制时会执行备份和清理程序；配置为0时，表示不限制大小。
+        </div>
+      </a-form-item>
       <a-form-item label="保留日志文件数" :label-col="labelCol" :wrapper-col="wrapperCol">
         <a-input-number v-model="config.app.keepLogFileCount" :step="1" :min="0" />
         <div class="form-help">
-          修改后，重启DS才生效，隔天才会清理多余的历史日志文件！
+          修改后，重启DS才生效，<code>隔天</code>或<code>达到日志大小限制</code>时，才会触发清理程序！
         </div>
       </a-form-item>
     </div>

